src/apps/events/admin.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import admin
+
+from src.apps.events.models import Event
+from src.apps.invitations.models import Invitation
+
+
+class InvitationInline(admin.TabularInline[Invitation, Invitation]):
```

Comment:
я не знаю для чего второй аргумент, можешь провести иследование? надо знать, подходит это или нет.

Reply:
`admin.TabularInline` - это класс, который используется в Django для создания встроенных форм для административного интерфейса. Когда вы указываете типы, как в `admin.TabularInline[Invitation, Invitation]`, вы указываете два типа: первый тип - это модель, для которой вы создаете встроенную форму, а второй тип - это модель, к которой вы создаете встроенную форму в качестве внешнего ключа.

Например, если у вас есть модели `Event` и `Invitation`, и каждое приглашение связано с определенным событием через внешний ключ, вы можете использовать `admin.TabularInline` для создания встроенной формы для приглашений, связанных с конкретным событием. В этом случае первый тип будет `Invitation`, а второй тип будет `Event`, чтобы указать связь между этими двумя моделями.

Таким образом, передача двух аргументов в `admin.TabularInline[Invitation, Event]` позволяет указать типы моделей, для которых создается встроенная форма и их связь.

Reply:
thanks mr.gpt, u got a nice russian
here is the answer i was looking for
```python
_ChildModelT = TypeVar("_ChildModelT", bound=Model)
_ParentModelT = TypeVar("_ParentModelT", bound=Model)

class InlineModelAdmin(Generic[_ChildModelT, _ParentModelT], BaseModelAdmin[_ChildModelT]):
    model: type[_ChildModelT]
    ...
    formset: type[BaseInlineFormSet[_ChildModelT, _ParentModelT, forms.ModelForm[_ChildModelT]]]
    ...
    parent_model: type[_ParentModelT]
    opts: Options[_ChildModelT]
```

so that was a stubs for `InlineModelAdmin` which is base class for Tabular Inline.
теперь можешь взять и еще добавить `parent_model = Event`

```python
class InvitationInline(admin.TabularInline[Invitation, Event]):
    model = Invitation
    parent_model = Event
    extra = 1
    readonly_fields = ("status",)
```